File: maui/KsDialogs.Maui/Contract/DialogLayoutArea.cs
namespace KsDialogs;

/// <summary>
/// サイズと位置の計算を行う基準領域。
/// </summary>
/// <remarks>
/// 水平・垂直の両軸に効く (core/ADR-0008)。
/// </remarks>
public enum DialogLayoutArea
{
    /// <summary>ダイアログを載せるウィンドウの全体。</summary>
    Window,

    /// <summary>ウィンドウからシステムバーなどが占める余白を控除した可視領域。</summary>
    VisibleArea,
}

File: android/ksdialogs-core/src/main/kotlin/jp/kamusoft/ksdialogs/DialogLayoutArea.kt
package jp.kamusoft.ksdialogs

/**
 * サイズと位置の計算を行う基準領域。
 *
 * 水平・垂直の両軸に効く (core/ADR-0008)。
 */
public enum class DialogLayoutArea {
    /** ダイアログを載せるウィンドウの全体。 */
    WINDOW,

    /** ウィンドウからシステムバーなどが占める余白を控除した可視領域。 */
    VISIBLE_AREA,
}
